types/index.d.ts

```diff
@@ -1,6 +1,6 @@
 export type { Action } from './Action.d.ts';
 
-export type { PlatformConfig, Config } from './Config.d.ts';
+export type { PlatformConfig, Config, LocalOptions, LogConfig } from './Config.d.ts';
```

Comment:
```suggestion
export * from './Config.d.ts';
```

Let's go all out and export everything, easier to maintain as well.

Reply:
Actually.... that makes them public API and I can't break them then :\ some of them are more meant as internal stuff,  so your way is better maybe